### ISSUE TITLE
Only warn about PHPUnit deprecations if on PHPUnit version with replacement

### ISSUE THREAD

**Issue by TimothyBJacobs** [kalessil/phpinspectionsea#1155]
First, thank you for reporting a bug or making a request. That takes time and we appreciate that!

Let's start with common details:
 
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Bug/False-Positive or Feature request                        |
| Plugin         | Php Inspections (EA Ultimate), 2.0.11 |
| Language level | PHP 7.1                        |

Current behaviour (description/screenshot:)
--
Using PHPUnit 6, I am warned that `assertEqualsWithDelta` is recommended since the `$delta` parameter for `assertEquals` will be deprecated in PHPUnit 8. However, the alternative is not available in the version of PHPUnit my project is using ( specified in composer.json ), so the warning is not helpful.

Expected behaviour
--
Not to warn about deprecations with no alternative for the current version.

Environment details
---
Information from Help -> About dialog (contains IDE and it's version details).

```
PhpStorm 2018.3.2
Build #PS-183.4886.46, built on December 19, 2018
Licensed to Timothy Jacobs
Subscription is active until November 5, 2019
JRE: 1.8.0_152-release-1343-b26 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.2
```

**Comment by kalessil** [comment]
Oh, it finally came time for introducing the PhpUnit version setting. Thank you for reporting @TimothyBJacobs.

**Comment by fruitl00p** [comment]
I'm running into this aswell... We're currently in a process of migrating to a later version of PHPUnit and the advisories given by the plugin dont align well with the (current) version we're actually using. Thus we're seeing a lot of warnings that we have to manually keep checking to see if some of them are 'correct' as others might result in false positives... (and breaking the builds ;))

It's not a very big deal, but is a hassle ;)

**Comment by kalessil** [comment]
Fixed!
